lib/ok.ex

```diff
@@ -64,9 +64,72 @@ defmodule OK do
   """
   def failure(reason), do: {:error, reason}
 
+  @doc """
```

Comment:
I personally think the best use case for the library is a fluent-style factory builder (as that is my personal use case). In this regard, you could consider adding a longer example that showcases the power of the concise syntax.